tests/FSharp.Build.UnitTests/FSharp.Build.UnitTests.fsproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net472;$(FSharpNetCoreProductTargetFramework)</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' == 'Unix'">$(FSharpNetCoreProductTargetFramework)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Unix' or '$(BUILDING_USING_DOTNET)' == 'true'">$(FSharpNetCoreProductTargetFramework)</TargetFrameworks>
```

Comment:
That's for easier local development in `FSharp.sln`.

Reply:
Yep, I meant it, typed the wrong thing 🤦

Reply:
No, it's FCS solution:
https://github.com/dotnet/fsharp/blob/318e1c85a73428e6267781aef867f9f290b8bc18/Directory.Build.props#L24-L26

This flag isn't supported with any other solutions.

Reply:
Alrighty, removed the change.

Reply:
The change was fine, but the comment was not right :)